### DISCUSSION TITLE
Request help: Cooperation invitation from Apache APISIX community

### DISCUSSION THREAD

**Discussion by lijing-21** [ory/hydra#3145] [Q&A] [answered]
Hi community,

I am Jing Li, from the Apache APISIX community. Apache APISIX is a Cloud-native API gateway, and it is the top-level project of the Apache Software Foundation. You can get more details from GitHub: [GitHub - apache/apisix: The Cloud-Native API Gateway](https://github.com/apache/apisix)

APISIX has a number of useful plug-ins. The "OpenID Connect Plug-in" for Apache APISIX supports the OpenID Connect protocol, and users can use this plug-in to enable Apache APISIX to interface with Ory Hydra services and be deployed as a centralized authentication gateway in the enterprise.

I am wondering if we could write a blog describing the steps to configure Ory Hydra authentication using the OpenID Connect plug-in for Apache APISIX. That will be a nice guide for our developers and will help Apache APISIX and Ory Hydra publicize. 

So if you are interested in writing this blog, welcome to discuss!

**Comment by aeneasr** [comment]
That would be awesome! @vinckr :)

**Reply by vinckr** [reply]
Hey @lijing-21 
I think its a great idea! 
Feel free to contact me in [our Slack](https://slack.ory.sh/) or via email to discuss next steps.

**Reply by lijing-21** [reply, marked as answer]
Hi @vinckr , sorry to be so late. We have completed this blog in the last past month.:https://apisix.apache.org/blog/2022/07/04/apisix-integrates-with-hydra

And I have contacted you in slack, looking forward to your reply!

**Reply by bangbangDong** [reply]
How do I configure route to configure openid-connect in version 3.0
